llvm/test/CodeGen/RISCV/rvv/vmsbf.ll

```diff
@@ -159,9 +159,9 @@ define <vscale x 8 x i1> @intrinsic_vmsbf_mask_m_nxv8i1_nxv8i1(<vscale x 8 x i1>
 ; CHECK:       # %bb.0: # %entry
 ; CHECK-NEXT:    vmv1r.v v10, v0
 ; CHECK-NEXT:    vmv1r.v v0, v9
-; CHECK-NEXT:    vsetvli zero, a0, e8, m1, tu, mu
+; CHECK-NEXT:    vsetvli zero, a0, e8, m1, ta, mu
 ; CHECK-NEXT:    vmsbf.m v10, v8, v0.t
-; CHECK-NEXT:    vmv1r.v v0, v10
+; CHECK-NEXT:    vmv.v.v v0, v10
```

Comment:
Is this a regression? Because vmv.v.v depends on vtype/vl, while vm1r.v doesn't.

Reply:
Probably depends on the microarchitecture. If VL is less than vlmax, then it can save copying some elements on some microarchitectures.

Reply:
Oh I see, I forgot it. Thanks!